code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm

```diff
@@ -238,7 +238,8 @@
 			sleep(duration/5)
 			if(!user || user.stat == DEAD)
 				return
-			command_announcement.Announce("Warning: Automatic countermeasures ineffective. Breach of primary network firewall imminent.", "Network Monitoring")
+			var/area/A = get_area(user)
+			command_announcement.Announce("Warning: Automatic countermeasures ineffective. Breach of primary network firewall imminent. Source of hack traced to [A.name].", "Network Monitoring")
```

Comment:
I am against this change as it punishes AIs that dont go the "stealth AI" route and immediately outs them to the entire station as this announces the source of the hack as AI core.